src/messages/models/Template/index.js

```diff
@@ -353,20 +353,58 @@ function createTemplateNode(options, markup) {
     prependText(disclaimer, toMarkup('disclaimer', mutationRules.disclaimer));
     appendImage(logoContainer, mutationRules.logo, 'PayPal Credit logo');
 
+    // Appends the message with the word "with" along with the product name. i.e. "with PayPal Credit"
+    const appendWithProductText = (insertPPText = false) => {
+        const [withText, productName] = getLocalProductName();
+        const span = document.createElement('span');
+        span.textContent = `${withText} `;
+
+        let el = 'strong';
+        switch (getLocaleClass()) {
+            case 'locale--GB':
+                el = 'span';
+                break;
+            default:
+                break;
+        }
```

Comment:
Consider simplifying this:
```suggestion
        const nodeType = getLocaleClass() === 'locale--GB' ? 'span' : 'strong';
```